core/domain/suggestion_services_test.py

```diff
@@ -1256,6 +1256,16 @@ def test_get_translation_suggestions_in_review_by_exploration(self):
             suggestions[1].status,
             suggestion_models.STATUS_IN_REVIEW)
 
+    def test_get_translation_suggestions_in_review_by_exploration_returns_empty_with_incorrect_language_code(self): # pylint: disable=line-too-long
```

Comment:
Maybe clearer if it was "returns_only_suggestions_with_supplied_language_code" or something similar?

Reply:
Makes sense, done!